docs/user-guide/deploy-on-bare-metal.md

```diff
@@ -38,91 +38,21 @@ _*Note*: This method could be used also in Virtual environment_
 
 ## iPXE
 
-iPXE deployment method
+*NOTE*: We use a sample URL, please change to fit your use case accordingly
```

Comment:
We can link to the [enhancement](https://github.com/openshift/assisted-service/blob/master/docs/enhancements/ipxe-host-boot.md) doc to show the URL format

Reply:
I think I'd rather just put the important details in here. Maybe how to get the ipxe download url since it's a separate endpoint.